hicommonwealth/commonwealth
# Remove version history

## Link to Issue
Closes: #7961

## Description of Changes
- Removes version history

## Test Plan
- Create thread, create comment, edit thread, edit comment
- Ensure that if the thread is created, you get "Published ..." and if the thread updated "Edited ..." on the thread page:
<img width="574" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/9f09a77c-9533-49da-a5a4-bb1ce53dddbc">

## Other Considerations
This PR removes version_history entirely. Note that we would lose the last ~7 days worth of version history data, as well as we would not be tracking future version history until a fix is pushed.
